reporter-web-app/src/App.css

```diff
@@ -183,13 +183,17 @@ tr.ant-table-expanded-row, tr.ant-table-expanded-row:hover {
 
 /* === Styling for TableView  === */
 
-/* Styling for 'Clear filters' button in TableView */
-.ort-table-operations button {
+/* Styling for 'Clear filters' and 'Shoe / Hide Columns' buttons in TableView */
+.ort-table-buttons .ant-btn-group {
     float: right;
-    margin: 16px 0px 16px 0px;
+    margin-bottom: 5px;
     z-index: 999;
 }
 
+.ort-table-toggle-columns .ant-dropdown-menu-item-selected {
```

Comment:
How about renaming `toggle-columns` to `package-filter` or `package-filter-menu` or `package-filter-selector`?

Reply:
A `filter` is Ant Design changes the number of row shown, whilst this option change the number of column visible.